cmd/referral/main.go

```diff
@@ -0,0 +1,202 @@
+	"github.com/Decentr-net/vulcan/internal/blockchain"
+	"github.com/Decentr-net/vulcan/internal/blockchain/rest"
+	"github.com/Decentr-net/vulcan/internal/health"
+	"github.com/Decentr-net/vulcan/internal/referral"
+	"github.com/Decentr-net/vulcan/internal/storage/postgres"
+)
+
+// nolint:lll,gochecknoglobals
+var opts = struct {
+	Postgres                   string `long:"postgres" env:"POSTGRES" default:"host=localhost port=5432 user=postgres password=root sslmode=disable" description:"postgres dsn"`
+	PostgresMaxOpenConnections int    `long:"postgres.max_open_connections" env:"POSTGRES_MAX_OPEN_CONNECTIONS" default:"0" description:"postgres maximal open connections count, 0 means unlimited"`
+	PostgresMaxIdleConnections int    `long:"postgres.max_idle_connections" env:"POSTGRES_MAX_IDLE_CONNECTIONS" default:"5" description:"postgres maximal idle connections count"`
+	PostgresMigrations         string `long:"postgres.migrations" env:"POSTGRES_MIGRATIONS" default:"migrations/postgres" description:"postgres migrations directory"`
+
+	BlockchainMainNode               string `long:"blockchain.main.node" env:"BLOCKCHAIN_MAIN_NODE" default:"http://zeus.testnet.decentr.xyz:26657" description:"decentr node address"`
+	BlockchainMainFrom               string `long:"blockchain.main.from" env:"BLOCKCHAIN_MAIN_FROM" description:"decentr account name to send stakes" required:"true"`
+	BlockchainMainTxMemo             string `long:"blockchain.main.tx_memo" env:"BLOCKCHAIN_MAIN_TX_MEMO" description:"decentr tx's memo'"`
+	BlockchainMainChainID            string `long:"blockchain.main.chain_id" env:"BLOCKCHAIN_MAIN_CHAIN_ID" default:"testnet" description:"decentr chain id"`
+	BlockchainMainClientHome         string `long:"blockchain.main.client_home" env:"BLOCKCHAIN_MAIN_CLIENT_HOME" default:"~/.decentrcli" description:"decentrcli home directory"`
+	BlockchainMainKeyringBackend     string `long:"blockchain.main.keyring_backend" env:"BLOCKCHAIN_MAIN_KEYRING_BACKEND" default:"test" description:"decentrcli keyring backend"`
+	BlockchainMainKeyringPromptInput string `long:"blockchain.main.keyring_prompt_input" env:"BLOCKCHAIN_MAIN_KEYRING_PROMPT_INPUT" description:"decentrcli keyring prompt input"`
+	BlockchainMainGas                uint64 `long:"blockchain.main.gas" env:"BLOCKCHAIN_MAIN_GAS" default:"10" description:"gas amount"`
+	BlockchainMainFee                string `long:"blockchain.main.fee" env:"BLOCKCHAIN_MAIN_FEE" default:"1udec" description:"transaction fee"`
+	BlockchainMainRESTNodeURL        string `long:"blockchain.main.rest_node_url" env:"BLOCKCHAIN_MAIN_REST_NODE_URL" default:"http://hera.mainnet.decentr.xyz" description:"REST endpoint URL"`
+
+	ReferralSenderReward   int `long:"referral.sender_reward" env:"REFERRAL_SENDER_REWARD" description:"referral sender reward uDEC'"`
+	ReferralReceiverReward int `long:"referral.receiver_reward" env:"REFERRAL_RECEIVER_REWARD" description:"referral receiver reward uDEC'"`
```

Comment:
maybe it's better to use sdk.Coin or sdk.Int

Reply:
ReferralSenderReward = 10 
ReferralReceiverReward = 10
are likely to be, there is no value to make them  sdk.Coin or sdk.Int